lib/bn.js

```diff
@@ -731,6 +731,45 @@ BN.prototype.uxor = function uxor(num) {
 };
 
 
+// Not ``this`` with ``width`` bitwidth
+BN.prototype.inotn = function inotn(width) {
+  assert(typeof width === 'number' && width >= 0);
+
+  var bytesNeeded = width / 26;
+
+  // Extend the buffer with leading zeroes
+  while (this.length < bytesNeeded) {
+    this.words[this.length++] = 0;
+  }
+
+  for (var i = 0; i < bytesNeeded; i++) {
+    var tmask = 0xfc000000; // top field
+    var bmask = 0x3ffffff;  // bottom field
+
+    if (width < 26) {
```

Comment:
Do you think we could "un-peel" this condition out of the loop? It seems to me that the if's body is executed only at the very end of the loop.